### PR TITLE
Add welcome to KNoT Cloud section

### PR DESCRIPTION
**Pull request by joaoaneto**

### CODE REVIEW

File: doc/cloud/cloud-welcome.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Welcome
+============
```
Comment (vitbaq) [inline doc/cloud/cloud-welcome.rst]: Fix section marker

File: doc/cloud/cloud-welcome.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+This solution makes it easy to build an IoT application that gathers, process, analyze and act on data generated by connected devices. A software development kit (`SDK <cloud-api.html>`_) is available to facilitate the application development by providing an easy to use API to the main services (device messaging, data storage and authenticator).
+
+----------------------------------------------------------------
+
+Features
+--------
+
+Distributed Messaging
+'''''''''''''''''''''''''''
```
Comment (vitbaq) [inline doc/cloud/cloud-welcome.rst]: Fix section marker

File: doc/cloud/cloud-welcome.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
+Welcome
```
Comment (vitbaq) [inline doc/cloud/cloud-welcome.rst]: Is this the better name for this section?

File: doc/cloud/cloud-welcome.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+'''''''''''''''''''''
+Some KNoT protocol characteristics were brought to the cloud protocol adapters to build a reliable model layer of device representation and supports scenarios in which devices can connect directly to the cloud through the network.
+
+----------------------------------------------------------------
+
+Architecture
+------------
+
+.. image:: ../../_static/cloud_overview_architecture.png
+   :alt: alternate text
```
Comment (vitbaq) [inline doc/cloud/cloud-welcome.rst]: Give an appropriate name for ":alt:"
- alt attribute specifies an alternate text for an image, if the image cannot be displayed.

Comment (joaoaneto) [reply doc/cloud/cloud-welcome.rst]: Done.

File: doc/cloud/cloud-welcome.rst (hunk 3 above)
Comment (joaoaneto) [reply doc/cloud/cloud-welcome.rst]: It was renamed to "Introduction".

File: doc/cloud/cloud-welcome.rst (hunk 2 above)
Comment (joaoaneto) [reply doc/cloud/cloud-welcome.rst]: Done.

File: doc/cloud/cloud-welcome.rst (hunk 1 above)
Comment (joaoaneto) [reply doc/cloud/cloud-welcome.rst]: Done.

**Pull request merged**